docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-okta-saml.md

```diff
@@ -60,9 +62,41 @@ Setting | Value
 
 :::note SAML Provider Caveats:
 
-- SAML Protocol does not support search or lookup for users or groups. Therefore, there is no validation on users or groups when adding them to Rancher.
+If you configure Okta without OpenLDAP, the following caveats apply due to the fact that SAML Protocol does not support search or lookup for users or groups.
+
+- There is no validation on users or groups when assigning permissions to them in Rancher.
 - When adding users, the exact user IDs (i.e. `UID Field`) must be entered correctly. As you type the user ID, there will be no search for other  user IDs that may match.
 - When adding groups, you must select the group from the drop-down that is next to the text box. Rancher assumes that any input from the text box is a user.
 - The group drop-down shows only the groups that you are a member of. You will not be able to add groups that you are not a member of.
 
 :::
+
+To enable searching for groups when assigning permissions in Rancher, you will need to configure a back end for the SAML provider that supports groups, such as OpenLDAP.
+
+## Setting up OpenLDAP in Rancher
+
+If you also configure OpenLDAP as the back end to Okta, it will return a SAML assertion to Rancher with user attributes that include groups. Then authenticated users will be able to access resources in Rancher that their groups have permissions for.
+
+### OpenLDAP Prerequisites
+
+If you use an Okta Directory as your IdP, you can setup an LDAP Interface for Rancher to use. See the [Okta Documentation](https://help.okta.com/en-us/Content/Topics/Directory/LDAP-interface-main.htm). You can also configure an external OpenLDAP server.
+
+Rancher must be configured with a LDAP bind account (aka service account) to search and retrieve LDAP entries pertaining to users and groups that should have access. It is recommended to not use an administrator account or personal account for this purpose and instead create a dedicated account in OpenLDAP with read-only access to users and groups under the configured search base (see below).
```

Comment:
OpenLDAP doesn't necessarily have the _concept_ of users. That's an optional schema; under the hood it's basically a tree-like structure and exceptionally malleable. Actual instructions for user creation will depend on the user's true backend. For example, Okta can provide an LDAP Interface and, if using this, the service account is just a regular user added to the Okda Directory.

Reply:
Ah, that's significantly more complicated than what I had in mind (simply linking the user to a pre-existing external page with instructions on how to create a dedicated account). But since we're specifically discussing Okta here, maybe we can link to this? https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-add-users.htm